website/docs/components/table/partials/code/component-api.md

```diff
@@ -24,6 +24,9 @@ The Table component itself is where most of the options will be applied. However
   <C.Property @name="<:body>" @type="named block">
     This is a named block where the content for the table body (`<tbody>`) is rendered.
     <Doc::ComponentApi as |C|>
+      <C.Property @name="[B].rowIndex" @type="number">
+        The value of the index asssociated with the `@each` loop.
```

Comment:
thought: I was thinking if we should mention that this is available only if `@model`/`@columns` exists, but that's true for most of the arguments